core/app/models/spree/payment_method/store_credit.rb

```diff
@@ -67,7 +67,7 @@ def credit(amount_in_cents, auth_code, gateway_options = {})
         originator = gateway_options[:originator]
 
         store_credit.credit(
-          amount_in_cents / 100.0.to_d,
+          ::Money.new(amount_in_cents, currency).to_d,
```

Comment:
Is the `.to_d` still required now that we're returning a `Fixnum` from `amount_in_cents`?